tasties_app/models.py

```diff
@@ -46,6 +55,65 @@ class UnitChoices(models.TextChoices, Enum):
     )
     description = models.CharField(max_length=64, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        return self.description + " " + str(self.amount) + " " + self.measurement_unit
```

Comment:
Won't we get: "Shallots, finely chopped 20 Gram"? Isn't that a bit... off? Compare with [here](https://www.allrecipes.com/recipe/141831/pan-seared-duck-breast-with-blueberry-sauce/#:~:text=2%20pounds%20bok%20choy%2C%20sliced). The whole syntactic processing is a bit of a heavy subject, we need to see how we're going to implement it. Maybe there's some external library we could use for that (but not right now)...

Reply:
Thanks, fixed